src/tools/generate-windows-sys/src/arm_shim.rs

```diff
@@ -0,0 +1,17 @@
+// Begin of ARM32 shim
+cfg_if::cfg_if! {
+if #[cfg(target_arch = "arm")] {
```

Comment:
The `if` only covers one target_arch, maybeb you don't need cfg_if at all.

Reply:
I just don't want to repeat the cfg guard twice (or even more in the future).